packages/dx-react-grid-material-ui/src/templates/paging-panel/pager.jsx

```diff
@@ -4,13 +4,11 @@ import classNames from 'classnames';
 import { withStyles } from 'material-ui/styles';
 import { PageSizeSelector } from './page-size-selector';
 import { Pagination } from './pagination';
-import { getBorderColor } from '../utils';
 
 const styles = theme => ({
   pager: {
     overflow: 'hidden',
     padding: theme.spacing.unit * 1.5,
-    borderTop: getBorderColor(theme),
```

Comment:
Because this border is 2 px

Reply:
This leads to the PaginPanel top border disappearance (see the featured-controlled-mode demo)